src/vs/workbench/contrib/terminal/browser/terminalMenus.ts

```diff
@@ -608,17 +609,16 @@ export function getTerminalActionBarArgs(target: TerminalLocation, profiles: ITe
 				id: contributed.id,
 				title
 			},
-			forceSplit: false,
-			target
+			location
 		})));
+		const splitLocation = location === TerminalLocation.Editor ? { viewColumn: SIDE_GROUP } : location;
```

Comment:
Double check this works for splitting in panel

Reply:
done